polytope/polytope.py

```diff
@@ -2212,7 +2212,8 @@ def region_diff(poly, reg, abs_tol=ABS_TOL, intersect_tol=ABS_TOL,
     M = np.sum(mi)
 
     if len(mi[0:len(mi)-1]) > 0:
-        csum = np.cumsum(np.vstack([0, mi[0:len(mi)-1]]))
+        logger.debug('mi: {}'.format(repr(mi)))
+        csum = np.cumsum(np.concatenate([[0], mi[0:len(mi)-1]]))
```

Comment:
However, it cannot be avoided, because a plain `0` raises `ValueError: zero-dimensional arrays cannot be concatenated`. This could be some minor motivation for keeping `numpy.hstack` (as in later versions of this line).

For posterity: `concatenate` uses `axis=0` by default, which works here because the arrays are 1-dimensional, and passing `axis=1` raises a warning `DeprecationWarning: axis != 0 for ndim == 1; this will raise an error in future versions of numpy` in `numpy == 1.7.1`.